infrastructure/modules/aks/law-collection.tf

```
@@ -77,6 +77,26 @@ resource "azurerm_monitor_data_collection_rule" "law" {
             "azure-arc"
           ],
           "enableContainerLogV2" : true
+
+          log_collection_settings = {
+            stdout = {
+              enabled            = false
+              exclude_namespaces = ["kube-system"]
+            },
+            stderr = {
+              enabled            = true
+              exclude_namespaces = ["kube-system", "monitoring", "linkerd-viz"]
```

linkerd-viz is not created by us as far as I know. Should probably remove it from this list
```suggestion
              exclude_namespaces = ["kube-system", "monitoring"]
```